cdap-ui/app/hydrator/services/create/stores/config-store.js

```diff
@@ -991,6 +991,9 @@ class HydratorPlusPlusConfigStore {
       this.$state.go('hydrator.create', this.$stateParams, {notify: false});
     }
     let config = this.getState();
+    if (config.__ui__.nodes.length === 0) {
```

Comment:
Hmm I tried replacing all `__ui__.nodes` with `config.stages`, but then a lot of things broke, since `__ui__.nodes` have some properties different from `config.stages`. Don't want to risk breaking changes in this PR...